### ISSUE TITLE
Schematic for `model.Rmd`

### ISSUE THREAD

**Issue by seabbs** [epinowcast/epinowcast#261]
As discussed in #258 it would be helpful to have a model schematic to give an overview of our current data-generating process and to help anchor the more detailed explanations of the various sections of the model. 

@adrian-lison has a version of this for his model which is quite nice and could form the basis/a starting point of our schematic. @adrian-lison what do you think? 

@nikosbosse has expressed interest in contributing something like this which would also be a helpful test of someone less involved in developing the data generating process interpreting what we are supporting. @nikosbosse do you think you are still able to do this? Ideally, we would target this at `0.3.0` which would mean resolving this issue in the next month or so.

**Comment by seabbs** [comment]
@adrian-lison could we borrow anything from you soon to be preprint here?

**Comment by nikosbosse** [comment]
That would be good - I tried coming up with something based on the Vignette and became overwhelmed with complexity...

**Comment by seabbs** [comment]
Yeah fair enough. I think having a few and starting simple is a good idea. `epidemia` docs are a good place to look for inspiration I think.

**Comment by seabbs** [comment]
Ping any and all. Anyone got anything cooking?

**Comment by nikosbosse** [comment]
I think it would be really useful to start with a collection of visualisations that we have at the moment. I recently tried starting from scratch solely based on the model Rmd file and it was daunting..
